app/i18n/it/admin.php

```diff
@@ -118,7 +118,7 @@
 		'empty_list' => 'Non ci sono estensioni installate',
 		'empty_list_help' => 'Controllare i log per determinare il motivo della lista estensioni vuota.',
 		'enabled' => 'Abilitata',
-		'is_compatible' => 'Is compatible',	// TODO
+		'is_compatible' => 'E\' compatibile',
```

Comment:
```suggestion
		'is_compatible' => 'E’ compatibile',
```

Favour typographic quotes

Reply:
So, in order to be perfect, we should use `È` (html `&Egrave;`). I'll review the files.

Reply:
Fixed.